### PR TITLE
Feat: update viewport doc

### PR DESCRIPTION
**Pull request by JujieX**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our [guidelines](https://github.com/galacean/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
update viewport doc

## Summary by CodeRabbit

- **Documentation**
  - Enhanced clarity and usability of the particle rendering documentation with updated titles, simplified headings, and improved table formatting.
  - Improved organization and presentation of the interface hierarchy documentation through section reordering and refined table formatting.
  - Modernized interactive viewport documentation with updated images, clearer table structures, and new attributes for enhanced user experience.
  - Introduced a new particle system management module example with GUI controls for real-time adjustments to particle effects in a 3D environment.

### CODE REVIEW

File: docs/zh/graphics/particle/renderer.md [hunk 1]
```diff
@@ -56,16 +56,15 @@ particleRenderer.setMaterial(material);
...
-| 播放（Play）    | 开始播放选中实体及其子节点上的粒子效果                             |
-| 速度（Speed）   | 调整当前播放速度                                                   |
-| 预览（Preview） | 选择播放选中实体及其子节点上的粒子效果，或者播放场景中所有粒子效果 |
+| 预览播放选项 | 释义                                       |
+| ------------ | ------------------------------------------ | --- |
+| 重播         | 停止当前的粒子效果播放，并立即从头开始播放 |
+| 停止         | 停止粒子效果的播放，并重置回初识状态       |
+| 暂停 播放    | 暂停 / 开始播放粒子效果                    |     |
+| 选中 / 全局  | 播放的是当前选中粒子，或者场景中全部粒子   |
+| 包围盒       | 当前选中粒子的包围盒                       |
```
Comment (cptbtptpbcptdtptp) [inline docs/zh/graphics/particle/renderer.md:67]: 表格格式有问题

File: docs/zh/interface/viewport.md [hunk 2]
```diff
@@ -34,52 +34,67 @@ label: Basics/Interface
...
+<img src="https://mdn.alipayobjects.com/huamei_qbugvr/afts/img/A*6IztTr2AERQAAAAAAAAAAAAADtKFAQ/original" alt="image-20240131190013320" style="zoom:50%;" />
+
+| 属性     | 内容                         |
+| :------- | :--------------------------- |
+| 拖动     | 自由拖动预览窗口             |
+| 定位     | 定位该相机在场景中的位置     |
+| 切换比例 | 切换不同设备、不同比例的窗口 |
+| 锁定     | 锁定相机预览窗口             |
+
+特别的，在层级树中带有相机组件的物体，能够通过直接同步视图的场景相机。
```
Comment (cptbtptpbcptdtptp) [inline docs/zh/interface/viewport.md:98]: 这句可以表达得更清楚一些。

**Review by cptbtptpbcptdtptp — APPROVED** [review]
+1

**Comment by JujieX** [comment]
in https://github.com/galacean/engine/pull/2283

**Pull request closed without merging**